package.json

```diff
@@ -78,7 +55,6 @@
         "wait-on": "^4.0.0"
     },
     "peerDependencies": {
-        "@dhis2/d2-i18n": "^1",
```

Comment:
Technically I think this should still be here?

Reply:
I think i18n is going away from this repo entirely, so it should go eventually. Not sure if it's ready for it in this state though, so will re-add.